package.json

```diff
@@ -8,7 +8,10 @@
     "ace3"
   ],
   "activationCommands": {
-    "atom-workspace": "language-arma-atom:toggle"
+    "atom-workspace": [
+      "language-arma-atom:toggle",
+      "language-arma-atom:open-rpt-file"
```

Comment:
Yes, it will ne automatically capitalized. I think you have no control to so this manually.